frontend/src/shared/forms/submit-btn/submit-btn.component.ts

```diff
@@ -10,11 +10,9 @@ export class SubmitBtnComponent {
   @Input() type?: string;
   @Input() disabled?: boolean;
 
-  @Output() click?: any = new EventEmitter<any>();
```

Comment:
Czemu usuwasz to? Pytam z ciekawości jak coś

Reply:
Bo nie jest narazie potrzebne, dodatkowo wykonywało akcję clicku podwójnie